src/agent/rustjail/src/container.rs

```diff
@@ -171,7 +171,7 @@ lazy_static! {
             r#type: String::from("c"),
```

Comment:
Not part of this PR, but I'm not sure why this device calls `String::from()` whereas the others call `.to_string()`.